src/OpenTK/Platform/X11/X11DisplayDevice.cs

```diff
@@ -192,7 +192,7 @@ private bool QueryXRandR(List<DisplayDevice> devices)
                         {
                             foreach (int depth in depths)
                             {
-                                available_res.Add(new DisplayResolution(0, 0, size.Width, size.Height, depth, (float)rate));
+                                available_res.Add(new DisplayResolution(0, 0, size.Width, size.Height, depth, (float)rate, new Vector2(size.Width * 25.4f / size.MWidth / 96.0f, size.Height * 25.4f / size.Height / 96.0f)));
```

Comment:
I think there is a bug here where you are dividing by `MWidth` for the X component, but you are not dividing by `MHeight` for the Y component.

Reply:
It would also be a lot nicer if you added some parenthesis to the math calculations so that you know the order the divisions are being done in.